### PR TITLE
Removing sqs sample from docs for now

### PR DESCRIPTION
**Pull request by jkotalik**
See https://github.com/project-radius/samples/issues/106, this sample had issues working that aren't regression from this release. We were debating disabling it from this release due to inconsistencies with the `name` property, we can bring it back once we get it working again.

### CODE REVIEW

**Comment by lakshmimsft** [comment]
Does the above link (https://wonderful-plant-020417a1e-448.westus2.azurestaticapps.net/) include this change? Cause I still see page for SQS there, so I guess no

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
----
+<!-- ---
 type: docs
 title: "Using AWS SQS Queues with Radius"
 linkTitle: "Using AWS SQS Queues"
 description: "Learn about how to model AWS SQS Queue resources in Bicep and them it in a Radius application"
 weight: 500
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-sqs/_index.md:6]:
```suggestion
weight: 500
toc_hide: true
hide_summary: true
```

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
----
+<!-- ---
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-sqs/_index.md]:
```suggestion
---
```

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 type: docs
 title: "Using AWS SQS Queues with Radius"
 linkTitle: "Using AWS SQS Queues"
 description: "Learn about how to model AWS SQS Queue resources in Bicep and them it in a Radius application"
 weight: 500
----
+--- -->
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-sqs/_index.md]:
```suggestion
---
```

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-sqs/_index.md:12]:
```suggestion

<!-- DISABLE_ALGOLIA -->

```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**